.travis.yml

```diff
@@ -2,22 +2,53 @@ sudo: true
 dist: bionic
 language: generic
 
+# We get this encrypted variable by:
+# 1) Setup the cache in google cloud storage as per https://docs.bazel.build/versions/master/remote-caching.html
+# 2) getting secure key as a `.json` from https://cloud.google.com/iam/docs/creating-managing-service-accounts#creating_a_service_account 
```

Comment:
What permissions did you give the service account? Any other tutorials you used to find all these steps?

Reply:
Unfortunately no. It was pretty easy to follow the bazel steps, but I don't think there's a super direct tutorial for this.